test/e2e/driver/azuredisk_csi_driver.go

```diff
@@ -26,35 +28,52 @@ import (
 	storagev1 "k8s.io/api/storage/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
 )
 
+const AzureDriverNameVar = "AZURE_STORAGE_DRIVER"
+
 // Implement DynamicPVTestDriver interface
 type azureDiskCSIDriver struct {
```

Comment:
Can we rename this whole driver to `AzureDiskDriver` since now it supports both csi and in-tree?

Reply:
Was prepping another PR for this. In any case, let merge and sent it.

Reply:
Done

Reply:
Thanks KK. The original comment was more to update all occurrences of `azureDiskCSIDriver` to `azureDiskDriver` including this filename.

Reply:
Thanks KK. To clarify, the original comment was to replace _all_ occurrences of `azureDiskCSIDriver` to `azureDiskDriver`including this filename.